apps/frontend/src/app/5_pages/LendPage/utils/LendingPoolDictionary.ts

```diff
@@ -0,0 +1,102 @@
+import { SupportedTokens } from '@sovryn/contracts';
+
+import { LendingPool } from './LendingPool';
+
+export class LendingPoolDictionary {
+  public static pools: Map<SupportedTokens, LendingPool> = new Map<
+    SupportedTokens,
+    LendingPool
+  >([
+    [
+      SupportedTokens.dllr,
+      new LendingPool(
+        'DLLR',
+        SupportedTokens.dllr,
+        [SupportedTokens.rbtc, SupportedTokens.bpro, SupportedTokens.sov],
+        false,
+        false,
+      ),
+    ],
+    [
+      SupportedTokens.rbtc,
+      new LendingPool(
+        'RBTC',
+        SupportedTokens.rbtc,
+        [
+          SupportedTokens.dllr,
+          SupportedTokens.xusd,
+          SupportedTokens.sov,
+          SupportedTokens.bpro,
+          SupportedTokens.doc,
+        ],
+        false,
+        false,
+      ),
+    ],
+    [
+      SupportedTokens.xusd,
+      new LendingPool(
+        'XUSD',
+        SupportedTokens.xusd,
+        [SupportedTokens.rbtc, SupportedTokens.bpro, SupportedTokens.sov],
+        true,
+        false,
+      ),
+    ],
```

Comment:
```suggestion
    [
      SupportedTokens.xusd,
      new LendingPool(
        'XUSD',
        SupportedTokens.xusd,
        [SupportedTokens.rbtc, SupportedTokens.bpro, SupportedTokens.sov],
        true,
        false,
      ),
    ],
```

Comment this out, only BTC and DLLR pools should be shown